pkg/patterns/declarative/reconciler.go

```diff
@@ -197,27 +197,7 @@ func (r *Reconciler) reconcileExists(ctx context.Context, name types.NamespacedN
 		return reconcile.Result{}, err
 	}
 
-	var newItems []*manifest.Object
-	for _, obj := range objects.Items {
-
-		unstruct, err := GetObjectFromCluster(obj, r)
-		if err != nil && !apierrors.IsNotFound(errors.Unwrap(err)) {
-			log.WithValues("name", obj.Name).Error(err, "Unable to get resource")
-		}
-		if unstruct != nil {
-			annotations := unstruct.GetAnnotations()
-			if _, ok := annotations["addons.k8s.io/ignore"]; ok {
```

Comment:
In original code, if some objects have annotation `addons.k8s.io/ignore`, that objects are skiped to append to the newItems.
And [`Apply`](https://github.com/kubernetes-sigs/kubebuilder-declarative-pattern/pull/163/files#diff-91d9000f2817ed2698caf7b294e34ed1bc739bf60b7dc1af3136055b18741312R223) doesn't apply these objects annotated.

Update on this PR changes that logic, because that logic move to after call `Apply`, I think.
WDYT?

Actually, this update fix the error noticed on #162, but that occurs only first reconcile loop as you said.

Reply:
Thanks for looking into the PR. I reviewed the logic again and yes, the order switch in this PR will affect the `addons.k8s.io/ignore` annotation. And for #162, it only occurs at the first round of reconciliation.

I could update the PR to just update the guestbook sample by adding the subresources. WDYD?

Reply:
Thank you :+1: